script.py

```diff
@@ -89,28 +85,24 @@ def print_vehicle_info(path):
 
 
 def is_valid_bus(user_input, response_data_json):
-    if(
-        user_input.lower() == 'all'
-        or user_input.lower() == 'exit'
-        or user_input.lower() == 'raw'
-    ):
-        return True
-
+    valid_inputs = ['all', 'exit', 'raw']
     for bus in response_data_json:
-        if bus['id'] == user_input:
-            return True
+        valid_inputs.append(bus['id'])
```

Comment:
Looks good. Just a bit nitpicky, you can use one-line style `for` loop here:

```python
valid_inputs += [bus['id'] for bus in response_data_json]
```

This can be used in available_buses as well.